spec/models/comment_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'validation' do
+    subject { Comment.new(text: 'Nice') }
+
+    before { subject.save }
+
+    it 'Comments text should contain text' do
+      subject.text = nil
+      expect(subject).to_not be_valid
+    end
+  end
+
+  describe 'Functionality' do
+    let(:user) do
+      User.create(
+        name: 'John Doe',
+        photo: 'https://upload.wikimedia.org/wikipedia/commons/a/ac/Default_pfp.jpg',
+        bio: 'Hello I am a test user',
+        posts_counter: 0
+      )
+    end
+
+    let(:post) do
+      Post.create(
+        title: 'Test Post',
+        text: 'This is a test post',
+        user:,
+        comment_counter: 0,
+        likes_counter: 0
+      )
+    end
+
+    subject do
+      Comment.new(
+        text: 'This is a test comment',
+        user:,
+        post:
+      )
+    end
+
+    describe 'Functionality' do
+      it 'updates the comments_counter of the post after saving' do
+        expect { subject.save }.to change { post.comment_counter }.by(1)
+      end
+    end
```

Comment:
- You did well testing if the `comments_counter` gets updated when a comment is saved but according to the project requirements, you are required to test all custom methods in your model, please test that the `comments_counter` increases when the `update_comment_counter` method is called. This way we can be sure that the method works correctly, please fix this 👍 👍